File: CPP/Code/Produk.cpp
#include <string>
#include <iostream>
using namespace std;

class Produk {

	//atribut pakenya private
	private:
		string nama;
		string kode;
		string merk;

	//method yang bersifat public
	public:

	    //constructor untuk inisiasi atribut
		Produk(string nama, string kode, string merk){
            this->nama = nama;
            this->kode = kode;
            this->merk = merk;
		}

		//getter nama
		void setNama(string nama) {
			this->nama = nama;
		}
		
		//setter nama
		string getNama() {
			return this->nama;
		}
		
		//getter kode
		void setKode(string kode) {
			this->kode = kode;
		}
		
		//setter kode
		string getKode() {
			return this->kode;
		}
		
		//getter merk
		void setMerk(string merk) {
			this->merk = merk;
		}
		
		//setter merk
		string getMerk() {
			return this->merk;
		}
		
		~Produk(){ }

};

File: PHP/Code/Produk.php
<?php
class Produk {

    private string $nama;
    private string $kode;
    private string $merk;

    public function __construct(string $nama, string $kode, string $merk) {
        $this->nama = $nama;
        $this->kode = $kode;
        $this->merk = $merk;
    }

    public function getNama(): string { 
        return $this->nama; 
    }
    public function setNama(string $nama): void { 
        $this->nama = $nama; 
    }

    public function getKode(): string { 
        return $this->kode; 
    }
    public function setKode(string $kode): void { 
        $this->kode = $kode; 
    }

    public function getMerk(): string {
        return $this->merk;
    }

    public function setMerk(string $merk): void {
        $this->merk = $merk;
    }
}
